default.nix

```diff
@@ -12,7 +12,7 @@ let
 in
 
   stdenv.mkDerivation {
-    name = "clever-cli-${version}";
+    name = "clever-tools-${version}";
 
     buildInputs = [ nodegit nodejs-6_x ];
```

Comment:
I just tried and everything run smoothly regardless of the nodejs-xx dependence. I guess we can remove it :P

Reply:
I'm testing on one of our project.